js/ui/components/polkitAgent.js

```diff
@@ -150,6 +153,16 @@ const AuthenticationDialog = new Lang.Class({
         this._passwordBox.add(this._passwordEntry,
                               { expand: true });
 
+        this._inputSourceManager = InputSourceManager.getInputSourceManager();
+        this._inputSourceIndicator = new InputSourceIndicator(this, false);
+        this._passwordBox.add(this._inputSourceIndicator.container);
+        this._manager = new PopupMenu.PopupMenuManager({ actor: this._inputSourceIndicator.container });
```

Comment:
See comment above about the popup menu manager not being needed outside of this function

Reply:
Still needs to be addressed